0.preprocessing_data/scripts/2a.make_z-stack_images.py

```diff
@@ -32,26 +43,51 @@
 # In[ ]:
 
 
-list_of_patients = [  # will be in a separate file in the future
-    "NF0014",
-    "NF0016",
-    "NF0018",
-    "NF0021",
-    "NF0030",
-    "NF0040",
-    "SARCO219",
-    "SARCO361",
-]
+bandicoot_path = pathlib.Path("/home/lippincm/mnt/bandicoot").resolve()
+if bandicoot_path.exists():
+    bandicoot = True
+else:
+    bandicoot = False
 
 
-# In[3]:
+# In[ ]:
+
+
+if bandicoot:
+    # comment out depending on whose computer you are on
+    # mike's computer
+    bandicoot_path = pathlib.Path("/home/lippincm/mnt/bandicoot").resolve()
+    # Jenna's computer
+    # bandicoot_path = pathlib.Path("/media/18tbdrive/GFF_organoid_data/")
```

Comment:
Same comment as before, you can change mine as they will reflect your paths ("home/jenna/mnt")